frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -172,28 +172,28 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
     type,
   });
 
-  const resultDataVolumeName = prefixedID(vmName, name);
+  const resultDataVolumeName = generateDataVolumeName(vmName, name);
 
   const resultVolume = VolumeWrapper.initializeFromSimpleData({
-    name,
+    name: resultDataVolumeName,
```

Comment:
mistake? resultDataVolumeName should be in typeData

Reply:
Definitely a mistake. Fixed!